bval-jsr/src/main/java/org/apache/bval/cdi/BValExtension.java

```diff
@@ -66,7 +70,7 @@
 public class BValExtension implements Extension {
     private static final Logger LOGGER = Logger.getLogger(BValExtension.class.getName());
 
-    private static final AnnotatedTypeFilter DEFAULT_ANNOTATED_TYPE_FILTER =
+    public static final AnnotatedTypeFilter DEFAULT_ANNOTATED_TYPE_FILTER =
```

Comment:
should stay private no? test should use it as default so no access needed

Reply:
made it public so i can cleanly reset it after the test ran again, I need to change this filter because otherwise the interceptor will not get enabled on the bean in my test

Reply:
in the context of tests you can copy `annotatedType -> !annotatedType.getJavaClass().getName().startsWith("org.apache.bval.")`
thing is that this should almost never be used cause it is wrong for all apps and only helpful in the context of this single build so it should be customized by apps